src/OpenTelemetry/Trace/Samplers/ParentOrElseActivitySampler.cs

```diff
@@ -0,0 +1,75 @@
+﻿// <copyright file="ParentOrElseActivitySampler.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
+using System.Diagnostics;
+
+namespace OpenTelemetry.Trace.Samplers
+{
+    /// <summary>
+    /// Sampler implementation which will take a sample if parent Activity or any linked Activity is sampled.
+    /// Otherwise, samples root traces according to the specified delegate sampler.
+    /// </summary>
+    public sealed class ParentOrElseActivitySampler : ActivitySampler
+    {
+        private readonly ActivitySampler delegateSampler;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ParentOrElseActivitySampler"/> class.
+        /// </summary>
+        /// <param name="delegateSampler">The <see cref="ActivitySampler"/> to be called to decide whether or not to sample a root trace.</param>
+        public ParentOrElseActivitySampler(ActivitySampler delegateSampler)
+        {
+            this.delegateSampler = delegateSampler ?? throw new ArgumentNullException(nameof(delegateSampler));
+
+            this.Description = $"ParentOrElse{delegateSampler.Description}";
+        }
+
+        /// <inheritdoc />
+        public override string Description { get; }
```

Comment:
Description MUST be ParentOrElse{delegateSampler.getDescription()}.

Reply:
Fixed. I had to stare at it for a minute. I was thinking... isn't that what I have? Added "{{" & "}}".

Reply:
:D Thanks!